sdk/src/main/java/io/radar/sdk/RadarLogger.kt

```diff
@@ -17,7 +17,7 @@ internal class RadarLogger(
         if (level >= RadarLogLevel.DEBUG) {
             Log.d(TAG, message, throwable)
 
-            Radar.sendLog(message)
+            Radar.sendLog(RadarLogLevel.DEBUG, message)
```

Comment:
Might be an unintended but neat side-effect product-wise. Given this is public, this means that developers will be able to send custom logs and they'll be cached + rendered on the Radar dashboard. 

Is this something we want to expose? cc @nickpatrick @corypisano 

e.g. 

```
Radar.sendLog(RadarLogLevel.DEBUG, "Jeff checked out")
```

This might also mean we have to be careful about PII

Reply:
@jkao the `sendLog` function is `internal`, so it can only be called from within the SDK.

Reply:
Ah you're right.